### PR TITLE
Add cb instruction shifts

### PR DESCRIPTION
**Pull request by ricfehr3**

### CODE REVIEW

**Comment by toastbucket** [comment]
its back 😎

File: src/cpu/test.rs [hunk 1]
```diff
@@ -922,6 +922,86 @@ fn test_rotate() {
...
+        Instruction::CbInstruction().as_byte(),
+        CbInstruction::SraMem().as_byte(),
+        Instruction::CbInstruction().as_byte(),
+        CbInstruction::SrlMem().as_byte(),
+        0x95,
+    ];
+
+    cpu.load_test_ram(&test_ram);
+
+    cpu.set_reg(Register8Bit::B, 0x95);
```
Comment (toastbucket) [inline src/cpu/test.rs:948]: use tests from manual :)

File: src/cpu/mod.rs [hunk 2]
```diff
@@ -349,6 +376,35 @@ impl Cpu {
         self.set_flag(Flag::C, bit0 == 1);
     }
 
+    fn shift_right(&mut self, regop: Register8Bit, clear_msb: bool) {
```
Comment (toastbucket) [inline src/cpu/mod.rs:379]: rename to `shit_right`

File: src/cpu/mod.rs [hunk 3]
```diff
@@ -296,6 +296,33 @@ impl Cpu {
         self.set_flag(Flag::C, bit7 == 1);
     }
 
+    fn shift_left(&mut self, regop: Register8Bit) {
```
Comment (toastbucket) [inline src/cpu/mod.rs:299]: rename to `shit_left`

**Review by toastbucket — CHANGES_REQUESTED** [review]

File: src/cpu/test.rs (hunk 1 above)
Comment (ricfehr3) [reply src/cpu/test.rs]: OUTDATED

File: src/cpu/mod.rs (hunk 2 above)
Comment (ricfehr3) [reply src/cpu/mod.rs:384]: `dookie_right`

File: src/cpu/mod.rs (hunk 3 above)
Comment (ricfehr3) [reply src/cpu/mod.rs:304]: `dookie_left`

**Review by toastbucket — APPROVED** [review]

**Pull request merged**